projects/libre-capital/index.js

```diff
@@ -111,72 +336,95 @@ async function getFundPrices() {
   navResults.output.forEach((result, i) => {
     if (result.success) {
       // navPerShare is in the first position of the returned tuple
-      priceMap[uniqueFundNames[i]] = Number(result.output[0]) / 1e6; // Assuming 18 decimals for NAV
+      priceMap[uniqueInstrumentID[i]] = Number(result.output[0]) / 1e6; // Assuming 18 decimals for NAV
     }
   });
 
   return priceMap;
 }
 
-async function polygonTvl({ polygon: block137 }) {
+async function ethTvl() {
   const balances = {}
   let totalTvl = 0;
   const fundPrices = await getFundPrices();
-  
-  // Get total supply of both receipt tokens
-  const supplies = await sdk.api.abi.multiCall({
-    abi: 'erc20:totalSupply',
-    calls: Object.values(RECEIPT_TOKENS.polygon).map(i => ({ target: i.address })),
-    chain: 'polygon',
-    block: block137,
-  })
-
-
-  // Map each token's total supply to represent RWA TVL
-  supplies.output.forEach((supply, i) => {
-    const token = Object.values(RECEIPT_TOKENS.polygon)[i]
-    const balance = supply.output;
-    const price = fundPrices[token.fundName] || 1;
-
-    // Convert balance to human readable and multiply by price
-    const adjustedBalance = Number(balance) / (10 ** token.decimals);
-    const valueUSD = adjustedBalance * price;
-    
-    totalTvl += valueUSD;
-  })
 
-  // Return the total value in the format DeFiLlama expects
+  // Query total supply for each token
+  for (const token of Object.values(RECEIPT_TOKENS.ethereum)) {
+    try {
+      const balance = await sdk.api.erc20.totalSupply({
+        target: token.address,
+      });
+
+      if (balance?.output) {
+        const price = fundPrices[token.instrumentId] || 1;
+        const adjustedBalance = Number(balance.output) / (10 ** token.decimals);
+        const valueUSD = adjustedBalance * price;
+        totalTvl += valueUSD;
+      }
+    } catch (e) {
```

Comment:
please remove these try/catch blocks - these will prevent our logging from showing errors when the adapter code fails

Reply:
addressed on [#0118163](https://github.com/DefiLlama/DefiLlama-Adapters/pull/13700/commits/01181636bceb5615f35963df04916b55bfd6ea76)